### PR TITLE
feat:Add yearsExperience to Applicant

### PR DESCRIPTION
**Pull request by jeyanthi-sm**
Added a new column named yearsExperience to the Applicant model.

### CODE REVIEW

File: src/api/applicant/content-types/applicant/schema.json [hunk 1]
```diff
@@ -66,6 +66,12 @@
       "type": "component",
       "repeatable": true,
       "component": "skills.skills"
+    },
+    "yearsExperience": {
+      "type": "decimal",
+      "required": true,
+      "min": 0,
+      "max": 100
```
Comment (becevka) [inline src/api/applicant/content-types/applicant/schema.json:74]: Hope humanity will find a lot of cure soon to fight aging and we will need to update this limit even further 😄

**Review by becevka — APPROVED** [review]

File: src/api/applicant/content-types/applicant/schema.json [hunk 2]
```diff
@@ -66,6 +66,12 @@
       "type": "component",
       "repeatable": true,
       "component": "skills.skills"
+    },
+    "yearsExperience": {
```
Comment (becevka) [inline src/api/applicant/content-types/applicant/schema.json:70]: You might want to name it `yearsOfExperience` or `experienceYears`.

File: src/api/applicant/content-types/applicant/schema.json [hunk 3]
```diff
@@ -66,6 +66,12 @@
...
+      "type": "decimal",
+      "required": true,
```
Comment (becevka) [inline src/api/applicant/content-types/applicant/schema.json:72]: Also note, in Strapi, required is validate on API level when the new records are created, not on database level. Thus existing `applicants` records will have this field empty, so you will need account for empty value in frontend.

**Review by ajaywayase — APPROVED** [review]

**Pull request merged**